common/src/Model/WorldNode.cpp

```diff
@@ -265,6 +265,22 @@ namespace TrenchBroom {
                     [&](BrushNode* brush)                      { m_nodeTree->insert(brush->physicalBounds(), brush); }
                 ));
             }
+
+            const auto updatePersistentId = [](auto* persistentNode, auto& nextPersistentId) {
+                if (const auto persistentLayerId = persistentNode->persistentId()) {
```

Comment:
Either here, or where the lambda is called, we should check if `persistentNode == defaultLayer()` and skip calling the lambda if so, because it's weird to assign it a persistent ID but then not actually write it out to the file.

Reply:
done! 8370ba53638ef0f1be99597ff3d7c362f7f001ad